# Comparing `tmp/odoo_addon_purchase_exception-16.0.1.0.0.4-py3-none-any.whl.zip` & `tmp/odoo_addon_purchase_exception-16.0.1.0.0.5-py3-none-any.whl.zip`

## zipinfo {}

```diff
@@ -1,29 +1,29 @@
-Zip file size: 33532 bytes, number of entries: 27
--rw-r--r--  2.0 unx     3093 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/README.rst
--rw-r--r--  2.0 unx       29 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/__init__.py
--rw-r--r--  2.0 unx      760 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/__manifest__.py
--rw-r--r--  2.0 unx     1608 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/data/purchase_exception_data.xml
--rw-r--r--  2.0 unx     6293 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/i18n/es.po
--rw-r--r--  2.0 unx     6229 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/i18n/fr.po
--rw-r--r--  2.0 unx     5795 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/i18n/purchase_exception.pot
--rw-r--r--  2.0 unx     5868 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/i18n/zh_CN.po
--rw-r--r--  2.0 unx       80 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/models/__init__.py
--rw-r--r--  2.0 unx      668 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/models/exception_rule.py
--rw-r--r--  2.0 unx     2210 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/models/purchase.py
--rw-r--r--  2.0 unx      805 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/models/purchase_line.py
--rw-r--r--  2.0 unx      152 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/readme/CONTRIBUTORS.rst
--rw-r--r--  2.0 unx      351 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/readme/DESCRIPTION.rst
--rw-r--r--  2.0 unx      202 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/security/ir.model.access.csv
--rw-r--r--  2.0 unx     9455 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/static/description/icon.png
--rw-r--r--  2.0 unx    12405 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/static/description/index.html
--rw-r--r--  2.0 unx       38 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/tests/__init__.py
--rw-r--r--  2.0 unx     5917 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/tests/test_purchase_exception.py
--rw-r--r--  2.0 unx     3670 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/views/purchase_view.xml
--rw-r--r--  2.0 unx       41 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/wizard/__init__.py
--rw-r--r--  2.0 unx      958 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/wizard/purchase_exception_confirm.py
--rw-r--r--  2.0 unx      511 b- defN 23-Mar-23 17:46 odoo/addons/purchase_exception/wizard/purchase_exception_confirm_view.xml
--rw-r--r--  2.0 unx     3711 b- defN 23-Mar-23 17:46 odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/METADATA
--rw-r--r--  2.0 unx       92 b- defN 23-Mar-23 17:46 odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/WHEEL
--rw-r--r--  2.0 unx        5 b- defN 23-Mar-23 17:46 odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/top_level.txt
-?rw-rw-r--  2.0 unx     2930 b- defN 23-Mar-23 17:46 odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/RECORD
-27 files, 73876 bytes uncompressed, 28528 bytes compressed:  61.4%
+Zip file size: 33772 bytes, number of entries: 27
+-rw-r--r--  2.0 unx     3093 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/README.rst
+-rw-r--r--  2.0 unx       29 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/__init__.py
+-rw-r--r--  2.0 unx      760 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/__manifest__.py
+-rw-r--r--  2.0 unx     1608 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/data/purchase_exception_data.xml
+-rw-r--r--  2.0 unx     6305 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/i18n/es.po
+-rw-r--r--  2.0 unx     6993 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/i18n/fr.po
+-rw-r--r--  2.0 unx     5795 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/i18n/purchase_exception.pot
+-rw-r--r--  2.0 unx     6632 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/i18n/zh_CN.po
+-rw-r--r--  2.0 unx       80 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/models/__init__.py
+-rw-r--r--  2.0 unx      668 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/models/exception_rule.py
+-rw-r--r--  2.0 unx     2210 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/models/purchase.py
+-rw-r--r--  2.0 unx      805 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/models/purchase_line.py
+-rw-r--r--  2.0 unx      152 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/readme/CONTRIBUTORS.rst
+-rw-r--r--  2.0 unx      351 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/readme/DESCRIPTION.rst
+-rw-r--r--  2.0 unx      202 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/security/ir.model.access.csv
+-rw-r--r--  2.0 unx     9455 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/static/description/icon.png
+-rw-r--r--  2.0 unx    12405 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/static/description/index.html
+-rw-r--r--  2.0 unx       38 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/tests/__init__.py
+-rw-r--r--  2.0 unx     5917 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/tests/test_purchase_exception.py
+-rw-r--r--  2.0 unx     3670 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/views/purchase_view.xml
+-rw-r--r--  2.0 unx       41 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/wizard/__init__.py
+-rw-r--r--  2.0 unx      958 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/wizard/purchase_exception_confirm.py
+-rw-r--r--  2.0 unx      511 b- defN 23-Mar-24 07:32 odoo/addons/purchase_exception/wizard/purchase_exception_confirm_view.xml
+-rw-r--r--  2.0 unx     3711 b- defN 23-Mar-24 07:32 odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/METADATA
+-rw-r--r--  2.0 unx       92 b- defN 23-Mar-24 07:32 odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/WHEEL
+-rw-r--r--  2.0 unx        5 b- defN 23-Mar-24 07:32 odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/top_level.txt
+?rw-rw-r--  2.0 unx     2930 b- defN 23-Mar-24 07:32 odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/RECORD
+27 files, 75416 bytes uncompressed, 28768 bytes compressed:  61.9%
```

## zipnote {}

```diff
@@ -63,20 +63,20 @@
 
 Filename: odoo/addons/purchase_exception/wizard/purchase_exception_confirm.py
 Comment: 
 
 Filename: odoo/addons/purchase_exception/wizard/purchase_exception_confirm_view.xml
 Comment: 
 
-Filename: odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/METADATA
+Filename: odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/METADATA
 Comment: 
 
-Filename: odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/WHEEL
+Filename: odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/WHEEL
 Comment: 
 
-Filename: odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/top_level.txt
+Filename: odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/top_level.txt
 Comment: 
 
-Filename: odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/RECORD
+Filename: odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/RECORD
 Comment: 
 
 Zip file comment:
```

## odoo/addons/purchase_exception/i18n/es.po

```diff
@@ -6,14 +6,15 @@
 msgstr ""
 "Project-Id-Version: Odoo Server 16.0\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2023-02-21 14:26+0000\n"
 "PO-Revision-Date: 2023-02-21 14:26+0000\n"
 "Last-Translator: \n"
 "Language-Team: \n"
+"Language: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: \n"
 
 #. module: purchase_exception
 #: model_terms:ir.ui.view,arch_db:purchase_exception.view_order_form
@@ -29,16 +30,15 @@
 #: model_terms:ir.ui.view,arch_db:purchase_exception.view_purchase_order_filter
 msgid "Blocked in Draft"
 msgstr "Bloqueado en presupuesto"
 
 #. module: purchase_exception
 #: model_terms:ir.ui.view,arch_db:purchase_exception.view_order_form
 msgid ""
-"Click here to be able to confirm this Agreement regardless of the "
-"exceptions."
+"Click here to be able to confirm this Agreement regardless of the exceptions."
 msgstr ""
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__create_uid
 msgid "Created by"
 msgstr "Creado por"
```

## odoo/addons/purchase_exception/i18n/fr.po

```diff
@@ -15,25 +15,36 @@
 "Language: fr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
 
 #. module: purchase_exception
+#: model_terms:ir.ui.view,arch_db:purchase_exception.view_order_form
+msgid "<strong>There are exceptions blocking this Purchase Order:</strong>"
+msgstr ""
+
+#. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_exception_rule__model
 msgid "Apply on"
 msgstr ""
 
 #. module: purchase_exception
 #: model_terms:ir.ui.view,arch_db:purchase_exception.view_purchase_order_filter
 #, fuzzy
 msgid "Blocked in Draft"
 msgstr "Bloquée en brouillon"
 
 #. module: purchase_exception
+#: model_terms:ir.ui.view,arch_db:purchase_exception.view_order_form
+msgid ""
+"Click here to be able to confirm this Agreement regardless of the exceptions."
+msgstr ""
+
+#. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__create_uid
 msgid "Created by"
 msgstr "Créé par"
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__create_date
 msgid "Created on"
@@ -47,32 +58,39 @@
 #. module: purchase_exception
 #: model:ir.model,name:purchase_exception.model_exception_rule
 #, fuzzy
 msgid "Exception Rule"
 msgstr "Règles d'exception"
 
 #. module: purchase_exception
-#: model:ir.ui.menu,name:purchase_exception.menu_purchase_exception_parent
+#: model:ir.model.fields,field_description:purchase_exception.field_purchase_order__exception_ids
 msgid "Exceptions"
 msgstr "Exceptions"
 
 #. module: purchase_exception
+#: model:ir.model.fields,field_description:purchase_exception.field_purchase_order__exceptions_summary
+msgid "Exceptions Summary"
+msgstr ""
+
+#. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__exception_ids
 #, fuzzy
 msgid "Exceptions to resolve"
 msgstr "Règles d'exception"
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__id
 msgid "ID"
 msgstr "ID"
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__ignore
+#: model:ir.model.fields,field_description:purchase_exception.field_purchase_order__ignore_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_order_line__ignore_exception
+#: model_terms:ir.ui.view,arch_db:purchase_exception.view_order_form
 #, fuzzy
 msgid "Ignore Exceptions"
 msgstr "Exceptions"
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm____last_update
 msgid "Last Modified on"
@@ -85,14 +103,19 @@
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__write_date
 msgid "Last Updated on"
 msgstr "Dernière mise à jour le"
 
 #. module: purchase_exception
+#: model:ir.model.fields,field_description:purchase_exception.field_purchase_order__main_exception_id
+msgid "Main Exception"
+msgstr ""
+
+#. module: purchase_exception
 #: model:exception.rule,description:purchase_exception.po_excep_no_email
 msgid "No Email for Vendor"
 msgstr "Pas de courriel pour le fournisseur"
 
 #. module: purchase_exception
 #: model:exception.rule,name:purchase_exception.po_excep_no_email
 msgid "No email on vendor"
@@ -146,37 +169,34 @@
 #, fuzzy
 msgid "Purchase order line"
 msgstr "Commande"
 
 #. module: purchase_exception
 #: model:ir.actions.server,name:purchase_exception.ir_cron_test_po_order_except_ir_actions_server
 #: model:ir.cron,cron_name:purchase_exception.ir_cron_test_po_order_except
-#: model:ir.cron,name:purchase_exception.ir_cron_test_po_order_except
 msgid "Purchase: Test Draft Orders Exception"
 msgstr "Achat : Test des exceptions des commandes en brouillon"
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_exception_rule__purchase_ids
 #, fuzzy
 msgid "Purchases"
 msgstr "Achat"
 
 #. module: purchase_exception
 #: model:exception.rule,name:purchase_exception.pol_excep_qty_check
 msgid "Quantity not positive"
 msgstr ""
 
-#. module: purchase_exception
-#: model_terms:ir.ui.view,arch_db:purchase_exception.view_order_form
-msgid ""
-"You have an outstanding\n"
-"                    exception to manage:"
-msgstr ""
-"Vous avez une exception\n"
-"                   non résolue à traiter :"
+#~ msgid ""
+#~ "You have an outstanding\n"
+#~ "                    exception to manage:"
+#~ msgstr ""
+#~ "Vous avez une exception\n"
+#~ "                   non résolue à traiter :"
 
 #~ msgid "Quantity not negative"
 #~ msgstr "Quantité non négative"
 
 #~ msgid "purchase.exception.confirm"
 #~ msgstr "purchase.exception.confirm"
```

## odoo/addons/purchase_exception/i18n/zh_CN.po

```diff
@@ -13,24 +13,35 @@
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: nplurals=1; plural=0;\n"
 "X-Generator: Weblate 3.8\n"
 
 #. module: purchase_exception
+#: model_terms:ir.ui.view,arch_db:purchase_exception.view_order_form
+msgid "<strong>There are exceptions blocking this Purchase Order:</strong>"
+msgstr ""
+
+#. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_exception_rule__model
 msgid "Apply on"
 msgstr "应用于"
 
 #. module: purchase_exception
 #: model_terms:ir.ui.view,arch_db:purchase_exception.view_purchase_order_filter
 msgid "Blocked in Draft"
 msgstr "在草稿中阻止"
 
 #. module: purchase_exception
+#: model_terms:ir.ui.view,arch_db:purchase_exception.view_order_form
+msgid ""
+"Click here to be able to confirm this Agreement regardless of the exceptions."
+msgstr ""
+
+#. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__create_uid
 msgid "Created by"
 msgstr "创建者"
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__create_date
 msgid "Created on"
@@ -43,31 +54,38 @@
 
 #. module: purchase_exception
 #: model:ir.model,name:purchase_exception.model_exception_rule
 msgid "Exception Rule"
 msgstr "异常规则"
 
 #. module: purchase_exception
-#: model:ir.ui.menu,name:purchase_exception.menu_purchase_exception_parent
+#: model:ir.model.fields,field_description:purchase_exception.field_purchase_order__exception_ids
 msgid "Exceptions"
 msgstr "异常"
 
 #. module: purchase_exception
+#: model:ir.model.fields,field_description:purchase_exception.field_purchase_order__exceptions_summary
+msgid "Exceptions Summary"
+msgstr ""
+
+#. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__exception_ids
 msgid "Exceptions to resolve"
 msgstr "解决的异常情况"
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__id
 msgid "ID"
 msgstr "ID"
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__ignore
+#: model:ir.model.fields,field_description:purchase_exception.field_purchase_order__ignore_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_order_line__ignore_exception
+#: model_terms:ir.ui.view,arch_db:purchase_exception.view_order_form
 msgid "Ignore Exceptions"
 msgstr "忽略异常"
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm____last_update
 msgid "Last Modified on"
 msgstr "最后修改时间"
@@ -79,14 +97,19 @@
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_purchase_exception_confirm__write_date
 msgid "Last Updated on"
 msgstr "最后更新时间"
 
 #. module: purchase_exception
+#: model:ir.model.fields,field_description:purchase_exception.field_purchase_order__main_exception_id
+msgid "Main Exception"
+msgstr ""
+
+#. module: purchase_exception
 #: model:exception.rule,description:purchase_exception.po_excep_no_email
 msgid "No Email for Vendor"
 msgstr "供应商没有Email"
 
 #. module: purchase_exception
 #: model:exception.rule,name:purchase_exception.po_excep_no_email
 msgid "No email on vendor"
@@ -137,35 +160,32 @@
 #: model:ir.model.fields.selection,name:purchase_exception.selection__exception_rule__model__purchase_order_line
 msgid "Purchase order line"
 msgstr "采购订单行"
 
 #. module: purchase_exception
 #: model:ir.actions.server,name:purchase_exception.ir_cron_test_po_order_except_ir_actions_server
 #: model:ir.cron,cron_name:purchase_exception.ir_cron_test_po_order_except
-#: model:ir.cron,name:purchase_exception.ir_cron_test_po_order_except
 msgid "Purchase: Test Draft Orders Exception"
 msgstr "购买：测试草稿订单异常"
 
 #. module: purchase_exception
 #: model:ir.model.fields,field_description:purchase_exception.field_exception_rule__purchase_ids
 msgid "Purchases"
 msgstr "采购"
 
 #. module: purchase_exception
 #: model:exception.rule,name:purchase_exception.pol_excep_qty_check
 msgid "Quantity not positive"
 msgstr ""
 
-#. module: purchase_exception
-#: model_terms:ir.ui.view,arch_db:purchase_exception.view_order_form
-msgid ""
-"You have an outstanding\n"
-"                    exception to manage:"
-msgstr ""
-"你有一个突出的\n"
-"                    异常需要管理："
+#~ msgid ""
+#~ "You have an outstanding\n"
+#~ "                    exception to manage:"
+#~ msgstr ""
+#~ "你有一个突出的\n"
+#~ "                    异常需要管理："
 
 #~ msgid "Quantity not negative"
 #~ msgstr "数量不是负数"
 
 #~ msgid "purchase.exception.confirm"
 #~ msgstr "purchase.exception.confirm"
```

## Comparing `odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/METADATA` & `odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/METADATA`

 * *Files 1% similar despite different names*

```diff
@@ -1,10 +1,10 @@
 Metadata-Version: 2.1
 Name: odoo-addon-purchase-exception
-Version: 16.0.1.0.0.4
+Version: 16.0.1.0.0.5
 Summary: Custom exceptions on purchase order
 Home-page: https://github.com/OCA/purchase-workflow
 Author: Akretion, Odoo Community Association (OCA)
 Author-email: support@odoo-community.org
 License: AGPL-3
 Platform: UNKNOWN
 Classifier: Programming Language :: Python
```

## Comparing `odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/RECORD` & `odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/RECORD`

 * *Files 6% similar despite different names*

```diff
@@ -1,15 +1,15 @@
 odoo/addons/purchase_exception/README.rst,sha256=8d-JSipJU5CaEekvz4GBPW87f5qM-B2_NY-EIq5VnuI,3093
 odoo/addons/purchase_exception/__init__.py,sha256=UxiV-7HaG7KrffYLF8b0ryDRo1uZekxEs0ohwXFwHsU,29
 odoo/addons/purchase_exception/__manifest__.py,sha256=vj9mQCU-1hVtWM1knEyqDHe0gfPZ5uuybeFPmp51fqo,760
 odoo/addons/purchase_exception/data/purchase_exception_data.xml,sha256=finiQ_0938XzLHcTL8g56KDpSv6axJmNSUeXb-s5Oko,1608
-odoo/addons/purchase_exception/i18n/es.po,sha256=74Yoms4U4dprzJk15jqHBRLp256MTp3m0Dgz0kXRZOE,6293
-odoo/addons/purchase_exception/i18n/fr.po,sha256=u4cuKMPOvC5pM878QgCO-uiLHQkX25lP26J3ZdcnAsM,6229
+odoo/addons/purchase_exception/i18n/es.po,sha256=1wKd_C6aIA5dhQd5EFxs5jB62ppvqpymfWpy60z5yko,6305
+odoo/addons/purchase_exception/i18n/fr.po,sha256=CneWYqBtr3FW3sMosDfIn27rCDWNNlI8W1g9dQS9U4g,6993
 odoo/addons/purchase_exception/i18n/purchase_exception.pot,sha256=G2Tt9_fDBwtxi4Jy2vBMLLosmWWhNWIsim5p-sevIAo,5795
-odoo/addons/purchase_exception/i18n/zh_CN.po,sha256=FZY95U8ywa3iFSf6LBy5G4Lnb_qGJwxci9dA0A8NoiU,5868
+odoo/addons/purchase_exception/i18n/zh_CN.po,sha256=mtZOhOo5Ktab1Pu3XDGN21KKnuwj0iHPcg0ohNHgRxg,6632
 odoo/addons/purchase_exception/models/__init__.py,sha256=JtiOgMQ8cYELwXLxtlFPq-rvcs7TeGD3vHtDARZQzP4,80
 odoo/addons/purchase_exception/models/exception_rule.py,sha256=OyqKpo8tYdjIeNolZ5mBhUzaMY16fH7j6TPIM1vQ9DM,668
 odoo/addons/purchase_exception/models/purchase.py,sha256=SXPVKtplzMUXRMczDsYhyq60YBkC8jHo1vPDngcgbJ4,2210
 odoo/addons/purchase_exception/models/purchase_line.py,sha256=2tRUUvsFGArCzz-ofoWdAhu92H_JRpdJXbtPjrY71ew,805
 odoo/addons/purchase_exception/readme/CONTRIBUTORS.rst,sha256=i1sjgJAlaPgG6IvJZjJgsNLWQHM4MYEBjBRHi9U09Fw,152
 odoo/addons/purchase_exception/readme/DESCRIPTION.rst,sha256=_Oh9b5htkmUdtVTtHkmYc61rFx6hq48URTGGgu-y6cY,351
 odoo/addons/purchase_exception/security/ir.model.access.csv,sha256=YjScvm7ke4ZW10OXqcqP_xuaxvcjR1VtGAZ2FXNntDM,202
@@ -17,11 +17,11 @@
 odoo/addons/purchase_exception/static/description/index.html,sha256=6lYm5uq_LH-R5O9UD1sIe4IaE_iTJPx1vn6GnSHC1e0,12405
 odoo/addons/purchase_exception/tests/__init__.py,sha256=j6gdli1ni2c_fpS0r3ELt3M216X70QdgUdNGAQfslIU,38
 odoo/addons/purchase_exception/tests/test_purchase_exception.py,sha256=OzGOrwLuOoKDR9hLsUFb6qBILRk2yyBxGPnsd61XV2M,5917
 odoo/addons/purchase_exception/views/purchase_view.xml,sha256=X2tVLXSJMw5qZGz62jJMHcJVvhmRw7NQ9pVg8K24Mi0,3670
 odoo/addons/purchase_exception/wizard/__init__.py,sha256=uvo1jP9USLl6BHCFegxFfXhkVDGPxdM8Y4a9FP-SIWc,41
 odoo/addons/purchase_exception/wizard/purchase_exception_confirm.py,sha256=JTPS6V54TbyT00GTclW_znETG4YoXq1sFrTdXeYNV1Y,958
 odoo/addons/purchase_exception/wizard/purchase_exception_confirm_view.xml,sha256=MzR0b7xaw0UkJZt1Jk9M_c8EGtYDgisA65m74VUxmMg,511
-odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/METADATA,sha256=NRNgw-y2JdaW8bqFEEpFHq2OprusnTu6O6APMeC0d4I,3711
-odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/WHEEL,sha256=2wepM1nk4DS4eFpYrW1TTqPcoGNfHhhO_i5m4cOimbo,92
-odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/top_level.txt,sha256=qBj40grFkGOfDZ2WDSw3y1RnDlgG0u8rP8pvGNdbz4w,5
-odoo_addon_purchase_exception-16.0.1.0.0.4.dist-info/RECORD,,
+odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/METADATA,sha256=-maspiepLd3UDEXc3ZWNLQCCNHCe8fd9R0qGltKqxF0,3711
+odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/WHEEL,sha256=2wepM1nk4DS4eFpYrW1TTqPcoGNfHhhO_i5m4cOimbo,92
+odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/top_level.txt,sha256=qBj40grFkGOfDZ2WDSw3y1RnDlgG0u8rP8pvGNdbz4w,5
+odoo_addon_purchase_exception-16.0.1.0.0.5.dist-info/RECORD,,
```

